app/models/katello/content_view.rb

```diff
@@ -579,6 +579,12 @@ def computed_modules_by_repoid
       PuppetModule.group_by_repoid(puppet_modules.flatten)
     end
 
+    def check_ready_to_import!
```

Comment:
Could you also check the file permissions here?

Reply:
Done! Its in the import action